scripts/generate_fishing_events

```diff
@@ -92,15 +94,15 @@ fi
 INSERT_SQL=${ASSETS}/bigquery/fishing-events.sql.j2
 echo "Inserting new records for ${START_DATE} to ${END_DATE}"
 jinja2 ${INSERT_SQL} \
-   -D messages=${SOURCE_TABLE//:/.} \
+   -D source_table=${SOURCE_TABLE//:/.} \
+   -D source_filter=${SOURCE_FILTER} \
    -D segment_vessel=${SEGMENT_VESSEL//:/.} \
    -D segment_info=${SEGMENT_INFO//:/.} \
    -D vessel_info=${VESSEL_INFO//:/.} \
-   -D dest=${DEST_TABLE//:/.} \
    -D start_yyyymmdd=$(yyyymmdd ${START_DATE}) \
    -D end_yyyymmdd=$(yyyymmdd ${END_DATE}) \
    -D min_duration=${MIN_DURATION} \
-   | bq query --max_rows=0
+   | bq query --max_rows=0 --allow_large_results --append_table --destination_table ${DEST_TABLE}
```

Comment:
@andres-arana why append-table? this will not make this bash idempotent. If you run it twice you will end up having duplicated entries?

Reply:
We are already deleting the records for the current date ranges beforehand in this same script. I don't know why we were using replace here.